tsfresh/feature_extraction/extraction.py

```diff
@@ -174,6 +176,44 @@ def _extract_features_parallel_per_kind(kind_to_df_map, settings, column_id, col
     return result
 
 
+def _extract_features_serial(kind_to_df_map, settings, column_id, column_value):
```

Comment:
Ok sure, it makes sense from a . I feel that we have a code duplication here. the main difference are the `map` versus `pool.imap_unordered` right? I want to prevent code duplication here. 

Do you think you can modify the `_extract_features_parallel_per_kind` method to do it in serial to use that for debugging?